sanic/request.py

```diff
@@ -64,7 +66,11 @@ def __init__(self, url_bytes, headers, version, method, transport):
     def json(self):
         if self.parsed_json is None:
             try:
-                self.parsed_json = json_loads(self.body)
+                if sys.version_info[:2] == (3, 5) and json_loads is json.loads:
```

Comment:
Instead of doing this, we should define `json_loads` after:

```python
def json_loads(data):
    return json.loads(data.decode())
```

Then we don't have to add the overhead of the version check and the try/except to every json access.